docs/reference/data-streams/logs.asciidoc

```diff
@@ -0,0 +1,49 @@
+[[logsds]]
+== Logs data stream
+
+You can use a logs data stream to store log data more efficiently. In our benchmarks,
+log data stored in a logs data stream used ~2.5 times less disk space than a regular data
+stream. The exact impact will vary per data set. The main benefit from using a logs data stream is reduced storage.
+
+The following features are enabled in a logs data stream:
+
+* <<synthetic-source,Synthetic source>>, which omits storing the _source. When _source or part of it is requested it is synthesized on the fly at runtime.
+
+* Index sorting. By default indices are sorted by `host.name` and `@timestamp` fields at index time, which can be overwritten.
+
+* Enable more space efficient compression for fields with doc values enabled.
+
+[discrete]
+[[how-to-use-logsds]]
+=== How to use a logs data stream
+
+Enabling a logs data stream only requires configuring the `index.mode` setting to `logs`. For example like this is done in this template:
+
+[source,console]
+----
+PUT _index_template/my-index-template
+{
+  "index_patterns": ["logs-*"],
+  "data_stream": { },
+  "template": {
+     "settings": {
+        "index.mode": "logs" <1>
+     }
+  },
+  "priority": 500 <2>
+}
+----
+// TEST
+
+<1> The index mode setting
+<2> Defining a priority to avoid other index templates with overlapping index patterns potentially overriding the settings.
```

Comment:
I'd suggest omitting this annotation because it's generic to all index templates

Reply:
By default ES does ship with a builtin template for logs-* with a priority of 100. If no priority is defined or below 100, then this template doesn't get picked up. Let me add this to the annotation explanation.